### PR TITLE
[tests] resolve sharp to a dummy small package to avoid install errors

### PR DESCRIPTION
**Pull request by EndangeredMassa**
Try setting:

```
"pnpm": {
  "overrides": {
    "gatsby-sharp>sharp": "npm:is-odd-num@1.1.1"
  }
},
```

to avoid install errors with `sharp`. We don't need it.

Uses [pnpm overrides](https://pnpm.io/package_json#pnpmoverrides).

### CODE REVIEW

**Review by TooTallNate — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -34,6 +34,11 @@
     "ts-jest": "28.0.5",
     "turbo": "1.7.4"
   },
+  "pnpm": {
+    "overrides": {
+      "gatsby-sharp>sharp": "npm:is-odd-num@1.1.1"
```
Comment (styfle) [inline package.json:39]: This package is actually quite large https://packagephobia.com/result?p=is-odd-num

Comment (styfle) [reply package.json:39]: I think this is going to lead to more confusion if someone ever tries to add a test for gatsby and expects sharp to work (for example `vc dev` tests)

**Review by styfle — CHANGES_REQUESTED** [review]
I think this is going to lead to more confusion if someone ever tries to add a test for gatsby and expects sharp to work (for example vc dev tests).

File: package.json (hunk 1 above)
Comment (cb1kenobi) [reply package.json:39]: Why `is-odd-num` needed to add `mocha` as a `dependency` instead of a `devDependency` is puzzling.

**Pull request closed without merging**